### PR TITLE
Header and auth screens

### PR DESCRIPTION
**Pull request by vijaytembugade**
- Added Login Page screen
- Added Signup Page screen
- Homepage Screen
- Header 
- Footer

### CODE REVIEW

File: src/Components/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          <a href="" className="nav-item">
+            Contact
+          </a>
+          <a href="" className="nav-item">
+            <img
+              className="avatar-sm"
+              src="https://miro.medium.com/fit/c/262/262/1*MFLNc70kh8oLPUsDaUH4sg.jpeg"
+            />
+          </a>
+        </div> */}
```
Comment (srishti-maurya) [inline src/Components/Header/Header.js:70]: you may delete these code if they are not required in future.

File: src/Pages/Homepage/Homepage.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            </strong>
+            !
+          </p>
+        </div>
+
+        <div className="flex-container homepage-desc-images">
+          <img src="/assets/Homepage/keypoints.svg" alt="" />
+          <img src="/assets/Homepage/Calender.svg" alt="" />
+          <img src="/assets/Homepage/taking-notes.svg" alt="" />
+        </div>
```
Comment (srishti-maurya) [inline src/Pages/Homepage/Homepage.js:36]: the `alt` attribute here is empty

**Review by srishti-maurya — APPROVED** [review]
UI looks clean and elegant

File: src/App.js [hunk 3]
```diff
@@ -1,7 +1,25 @@
...
+import Login from "./Pages/Login/Login";
+import Signup from "./Pages/Signup/Signup";
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Header />
+      <div className="main-container">
+        <Routes>
```
Comment (MarkVed17) [inline src/App.js:14]: Suggestion: you can create a routes component

File: src/Components/LoginForm/LoginForm.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        <h2>Login</h2>
+
+        <label>
+          <span className="input-label"> Email </span>
+          <input required type="email" />
+        </label>
+
+        <label>
+          <span className="input-label"> Password </span>
+          <input required minlength="5" type="password" />
```
Comment (MarkVed17) [inline src/Components/LoginForm/LoginForm.js]: minLength*

File: src/Components/SignupForm/SignupForm.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        <h2>Sign in</h2>
+
+        <label>
+          <span class="input-label"> Email </span>
+          <input required type="email" />
+        </label>
+
+        <label>
+          <span class="input-label"> Password </span>
+          <input required minlength="5" type="password" />
```
Comment (MarkVed17) [inline src/Components/SignupForm/SignupForm.js]: minLength*

File: src/Components/SignupForm/SignupForm.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        </label>
+        <label>
+          <span class="input-label"> Comfirm Password </span>
+          <input required minlength="5" type="password" />
```
Comment (MarkVed17) [inline src/Components/SignupForm/SignupForm.js]: here too

**Review by MarkVed17 — APPROVED** [review]
The UI looks good. Code readability is great as well. Good to merge.

File: src/Components/Header/Header.js (hunk 1 above)
Comment (vijaytembugade) [reply src/Components/Header/Header.js:70]: This code is needed afterwords so I kept it.

File: src/Pages/Homepage/Homepage.js (hunk 2 above)
Comment (vijaytembugade) [reply src/Pages/Homepage/Homepage.js:36]: I will add it

**Review by Nikhil-Nirmal — APPROVED** [review]

**Pull request merged**